rpms/pulp-scl-python-django/pulp-scl-python-django.spec

```diff
@@ -0,0 +1,60 @@
+%{?scl:%scl_package python-django}
+%{!?scl:%global pkg_name %{name}}
+
+# Fix this build.  TODO: set this in pulp-scl properly
+%global python_version 3.5
+
+%global srcname Django
```

Comment:
Talked to pcreech. This needs to be global because the build needs to access srcname